### PR TITLE
update sali1 to sali

### PR DESCRIPTION
**Pull request by umjnega**
I think this is what albert is asking for.  in the TDX ticket.

### CODE REVIEW

File: manifests/profile/www_lib/vhosts/apps_lib.pp [hunk 1]
```diff
@@ -231,7 +231,7 @@
     ssl_proxy_machine_cert        => $client_cert,
 
     custom_fragment               => @(EOT)
-      ProxyPassReverse / https://sali1.lib.umich.edu:8443/
+      ProxyPassReverse / https://sali.lib.umich.edu:8443/
     | EOT
```
Comment (skorner) [inline manifests/profile/www_lib/vhosts/apps_lib.pp:235]: We shouldn't need any of lines 228-235, given we are not proxying to an app anymore.

File: manifests/profile/www_lib/vhosts/apps_lib.pp [hunk 2]
```diff
@@ -192,7 +192,7 @@
       },
       {
         rewrite_cond => '%{REQUEST_URI} !^/openid-connect',
```
Comment (skorner) [inline manifests/profile/www_lib/vhosts/apps_lib.pp:194]: This condition can be removed now. It's probably also best to add a comment about what we are redirecting to (a standalone drupal-based SALI.

File: manifests/profile/www_lib/vhosts/apps_lib.pp
Comment (skorner) [inline manifests/profile/www_lib/vhosts/apps_lib.pp:1]: Lines 144-164 are also no longer needed.

**Review by skorner — CHANGES_REQUESTED** [review]
I left some notes about more we can/should cleanup. The changes should probably be tested in a puppet env on macc-www-lib-testing first with Albert verifying the behavior is as expected (he'll need a local hosts alias of apps.lib to 141.213.128.250 to test).

File: manifests/profile/www_lib/vhosts/apps_lib.pp (hunk 1 above)
Comment (umjnega) [reply manifests/profile/www_lib/vhosts/apps_lib.pp:235]: Ah, ok I misunderstood.  I Thought Albert was asking that we proxy back to sali.lib now that its hosted in AWS.

File: manifests/profile/www_lib/vhosts/apps_lib.pp (hunk 2 above)
Comment (umjnega) [reply manifests/profile/www_lib/vhosts/apps_lib.pp]: Removed and comment added.

File: manifests/profile/www_lib/vhosts/apps_lib.pp
Comment (umjnega) [reply manifests/profile/www_lib/vhosts/apps_lib.pp:1]: removed this section as well.

**Comment by umjnega** [comment]
If this passed,  I can apply it to macc-www-lib-testing and see if albert can test it.

File: manifests/profile/www_lib/vhosts/apps_lib.pp [hunk 3]
```diff
@@ -190,10 +169,8 @@
         # 2016-08-29 skorner per nancymou
         rewrite_rule => '^/islamic	http://guides.lib.umich.edu/islamicmss/find 	[redirect=permanent,last]'
       },
-      {
-        rewrite_cond => '%{REQUEST_URI} !^/openid-connect',
-        rewrite_rule => '^(/instruction/request.*)$ https://sali1.lib.umich.edu:8443$1 [P]',
-      },
+
+        # No More SALI redirect in favor of stand-alone drupal app
```
Comment (skorner) [inline manifests/profile/www_lib/vhosts/apps_lib.pp:173]: We still need to include a basic redirect to the new site at the former URL root:
`rewrite_rule => '^(/instruction/request.*)$ https://sali.lib.umich.edu/ [redirect=permanent,last]'`

**Review by skorner — APPROVED** [review]
That's great news. This all looks good to me. Feel free to merge next week after squashing to 1 commit and for good measure changing the PR title from "update sali1 to sali" to something like "replace apache config for locally hosted sali with a redirect".

**Pull request merged**